templates/tab/js/default/public/auth-end.html

```diff
@@ -12,49 +12,40 @@
       integrity="sha384-mhp2E+BLMiZLe7rDIzj19WjgXJeI32NkPvrvvZBrMi5IvWup/1NUfS5xuYN5S3VT"
       crossorigin="anonymous"
     ></script>
+    <script type="text/javascript" src="https://alcdn.msauth.net/browser/2.21.0/js/msal-browser.min.js"></script>
```

Comment:
we may add integrity to it, not sure whether crossorigin is required. Please help investigate.

Reply:
Fixed. Thx!